tests/integration/test_comprehensive_workflow.py

```
@@ -0,0 +1,707 @@
+            assert "error" in result_data or "results" in result_data
+
+
+@pytest.mark.asyncio
+class TestDataIntegrity:
+    """Test data integrity across the workflow."""
+
+    async def test_metadata_consistency(self, sample_documentation_files):
+        """Test metadata consistency across extraction pipeline."""
+        for doc_file in sample_documentation_files:
+            # Extract frontmatter
+            frontmatter, content = extract_frontmatter(doc_file["content"])
```

Pylint (3.3.7)

[warning] Unused variable 'content'

(W0612)